Makefile

```diff
@@ -40,16 +31,33 @@ CCM_E2E_ARGS ?= -ginkgo.skip=\\[Serial\\]\\[Slow\\]
 #The test args for Kubernetes e2e tests
 TEST_E2E_ARGS ?= '--ginkgo.focus=Port\sforwarding'
 
+# Generate all combination of all OS, ARCH, and OSVERSIONS for iteration
+# golang/strecth supports only amd64, arm32v7, arm64v8 and i386, before we have an
+# explicit requirement for other arch support, let's keep golang/stretch
+# https://github.com/docker-library/official-images/blob/master/library/golang
+ALL_ARCH.linux = amd64 arm arm64
```

Comment:
We had already upgraded to [golang/buster](https://github.com/kubernetes-sigs/cloud-provider-azure/blob/master/cloud-node-manager.Dockerfile#L17). The comment seems obsolete.

Reply:
Thanks for the note.